Check for ansible-core version update during a major release (#52)
ansible-collections/cloud-content-handbook: Releases/release_collections.md

   * For minor changes, increment y. Resulting branch will be x.y+1.0.
   * For bugfixes/patch/trivial changes, increment z. Resulting branch will be x.y.z+1.

> * For a **major release**, switch to the new branch stable-X, where X represents the major version number. This branch is typically created during the code freeze. If it doesn’t already exist, create it from the main branch at the time of release.

Reviewer: ```suggestion
* For a **major release**, switch to the new branch `stable-X`, where `X` represents the major version number. This branch is typically created during the code freeze. If it doesn’t already exist, create it from the `main` branch at the time of release.
```

   ```
   git checkout main
   git branch stable-X main  --> If a new branch is needed
   git checkout stable-X
   ```

> * For a **major release**, if an ansible-core version(`requires_ansible`) update is not already included, please verify whether an update is needed. If it is, create a separate PR for the version update and ensure it is merged to the Stable-X branch before starting the release preparation PR. If you're unsure whether an update is required, please consult the team in the team channel.

Reviewer: ```suggestion
* For a **major release**, if an ansible-core version (`requires_ansible`) update is not already included, please verify whether an update is needed. If it is, create a separate PR for the version update and ensure it is merged to the `stable-X` branch before starting the release preparation PR. If you're unsure whether an update is required, please consult the team in the team Slack channel.
```

> * For a **minor release**, make sure the backport of the PRs from the `main` branch to the release branch is successful. For backporting of PRs refer to [the backporting documentation](https://github.com/ansible-collections/cloud-content-handbook/blob/main/backport_changes.md).

Reviewer: ```suggestion
* For a **minor release**, make sure the backport of the PRs from the `main` branch to the release branch is successful. For more information on backporting PRs, refer to [the backporting page](https://github.com/ansible-collections/cloud-content-handbook/blob/main/backport_changes.md) of the team handbook.
```

* For a minor release, make sure the backport of the PRs from the `main` branch to the release branch is successful. For backporting of PRs refer to [the backporting documentation](https://github.com/ansible-collections/cloud-content-handbook/blob/main/backport_changes.md).

> * For a major release, if an ansible-core version(`requires_ansible`) update is not already included, please verify whether an update is needed. If it is, create a separate PR for the version update and ensure it is merged before starting the release preparation PR. If you're unsure whether an update is required, please consult the team in the team channel.

Reviewer 2: Just clarifying, do we want to do this before or after creating the new branch off of main?

Author: Thanks for catching that. Ideally, this should be done after the stable-X branch is created during the code freeze. This PR needs to be merged into the stable-X branch. I’ve updated the process with more clarity; let me know what you think.
